src/lib/precondition.ts

```diff
@@ -178,7 +178,7 @@ To write a correct circuit, you must avoid any dependency on the concrete value
   let value: U;
   if (accountOrNetwork === 'account') {
     let address = party.body.publicKey;
-    let account: any = Mina.getAccount(address);
+    let account: any = Mina.getAccount(address, getDefaultTokenId());
```

Comment:
we probably want to handle other token ids on the zkapp account, in places like this? if yes, it would be good to track it somehow